packages/core/src/server/service.ts

```diff
@@ -164,12 +163,17 @@ export class ServerService {
         return res.status(503).json(result);
       }
 
-      return graphqlMiddleware(req, res);
+      return graphqlMiddleware(req, res, next);
     });
 
     // NOTE: Deprecating use of root endpoint for GraphQL queries in favor of /graphql.
     // This will be removed in a future release.
-    this.app?.use("/", graphqlMiddleware);
+    this.app?.use("/", (_, response) => {
+      response.setHeader("Content-Type", "text/html");
+      response.send(
+        graphiQLPage({ endpoint: "http://localhost:42069/graphql" })
+      );
+    });
```

Comment:
See [this PR for context](https://github.com/0xOlias/ponder/pull/360).

Given that context, I think what we actually need to do here is update BOTH the `/` and `/graphql` endpoints to serve GET requests with GraphiQL and POST requests using the `graphql-http` middleware. The only difference is that `/graphql` returns a `503` for both GET and POST requests until historical indexing is complete.

Reply:
Oh yeah my plan was to refactor this in a separate PR and basically handle the comment right above this. So GET goes to playground, POST is request, etc. So I plan to do that right after this in another pr if that works.